packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol

```diff
@@ -450,8 +425,8 @@ contract AssetMappings is IAssetMappings, Initializable{
      * @dev Gets the configuration paramters of the reserve
      * @param asset Address of asset token you want params for
      **/
-    function getParams(address asset)
-        external view
+    function getParams(address asset, uint64 trancheId)
+        external view override
         returns (
             uint256 baseLTV,
```

Comment:
no need for named params since we're not using them

Reply:
It helps during tests to choose the exact param